src/sentry/api/endpoints/organization_dynamic_sampling_sdk_versions.py

```diff
@@ -157,18 +164,19 @@ def get(self, request: Request, organization) -> Response:
                     "project": project,
                     "latestSDKName": sdk_name,
                     "latestSDKVersion": sdk_version,
-                    "isSendingSampleRate": bool(row[f"equation|{avg_equation}"]),
+                    "isSendingSampleRate": bool(row[f"equation|{avg_sample_rate_equation}"]),
+                    "isSendingSource": bool(row[f"equation|{avg_transaction_source_equation}"]),
                 }
 
         # Essentially for each project, we fetch all the SDK versions from the previously
         # computed dictionary, and then we find the latest SDK version according to
         # semantic versioning and return the info for that particular project SDK version.
         project_info_list = [
             project_to_sdk_version_to_info_dict[project][
-                sorted(
-                    list(project_to_sdk_version_to_info_dict[project].keys()),
+                max(
```

Comment:
it's unrelated to this PR, but until i'm here i feel `max` is better than `sorted()[-1]`.

Reply:
`max` can be slower than `sorted` since it's a generic function, though we'd have to time this. Using `max` here is certainly more semantic. Unless there was a strong reason to go with `sorted`, this looks good.

Reply:
@jan-auer  you are right, but based on [SO](https://stackoverflow.com/questions/35014951/why-is-max-slower-than-sort) you can see `a.sort()[-1]` is faster than max, but we do here `sorted()[-1]`.

```
python -m timeit -s 'import random;a=list(range(10000));random.shuffle(a)' 'a.sort(); a[-1]'
10000 loops, best of 5: 24 usec per loop
```

```
python -m timeit -s 'import random;a=list(range(10000));random.shuffle(a)' 'max(a)'
5000 loops, best of 5: 91.7 usec per loop
```

```
python -m timeit -s 'import random;a=list(range(10000));random.shuffle(a)' 'sorted(a)[-1]'
500 loops, best of 5: 874 usec per loop
```

So if we really want to do it faster: we should go with `a.sort(); a[-1]`.